stable/database/README.md

```diff
@@ -307,6 +307,19 @@ The following tables list the configurable parameters for the `database` option
 | ----- | ----------- | ------ |
 | `nuodb.config` | [NuoDB database options][6] | `nil` |
 
+#### restore.*
```

Comment:
It looks like this section is specific to restore chart. Why we are documenting it in the database chart readme?

Reply:
We need to decide on this point.

A number of the chart READMEs seem to be a copy of the others - so that causes sections not covered directly by a specific chart to be documented in the README for that chart.

It is certainly true that all values are _valid_ in any chart, but there is a very strong case for only documenting values for a chart if that value is used by that chart.

Reply:
I was thinking about it this way:
- if a chart uses a value, it should be documented in the README.

These documents should be fairly standalone. When you wanna install via Rancher, you can scroll in the top window and get a quick looksie at this doc and it really helps to have everything there.